cypress/integration/hw_15.js

```diff
@@ -0,0 +1,426 @@
+import * as data from './consts.js';
+import { DATA_LOCATORS } from './locators.js'; 
+import * as func from './funcs.js';
+
+const { _ } = Cypress;
+
+describe('Creating new user and verification', () => {
+    before (()=> {
+    cy.visit("/webtables")});
+
+it('Filling data in the "First Name" field  and verification', () => {
+    cy.contains('Add').click()
+    cy.get(DATA_LOCATORS.firstName).type(data.RANDOM_NAME)
+    .should("have.value", data.RANDOM_NAME);
+});
+
+it('Filling data in the "Last Name field" and verification', () => {
+    cy.get(DATA_LOCATORS.lastName).type(data.RANDOM_LAST_NAME)
+    .should("have.value", data.RANDOM_LAST_NAME);
+});
+
+it('Filling data in the "E-mail" field and verification', () => {
+    cy.get(DATA_LOCATORS.userEmail).type(data.RANDOM_EMAIL)
+    .should("have.value", data.RANDOM_EMAIL);
+});
+
+it('Filling data in the "Age" field and verification', () => {
+    cy.get(DATA_LOCATORS.userAge).type(data.RANDOM_AGE)
+    .should("have.value", data.RANDOM_AGE);
+});
+
+it('Filling data in the "Salary" field and verification', () => {
+    cy.get(DATA_LOCATORS.salary).type(data.RANDOM_NUM)
+    .should("have.value", data.RANDOM_NUM);
+});
+
+it('Filling data in the "Department" field and verification', () => {
+    cy.get(DATA_LOCATORS.department).type(data.RANDOM_ADDRESS)
+    .should("have.value", data.RANDOM_ADDRESS);
+});
+
+it('Submitting data', () => {
+    cy.get(DATA_LOCATORS.submit).click()
+});
+
+it('Checking the correctness of all data', () => {
+    cy.contains(data.RANDOM_NAME);
+    cy.contains(data.RANDOM_LAST_NAME);
+    cy.contains(data.RANDOM_EMAIL);
+    cy.contains(data.RANDOM_AGE);
+    cy.contains(data.RANDOM_NUM);
+    cy.contains(data.RANDOM_ADDRESS)
+});
```

Comment:
cy.contains() it is not  the verification in cypress. Use withing() for applying a lot of verification and .should() for verification